cf-agent/verify_files_utils.c

```diff
@@ -591,11 +586,8 @@ static PromiseResult CfCopyFile(EvalContext *ctx, char *sourcefile,
                          sourcefile, destfile);
                     result = PromiseResultUpdate(result, PROMISE_RESULT_FAIL);
                 }
-
-                return result;
```

Comment:
Do you think this early `return` should really be removed? It might make things easier to read when there's a lot of code below like in this case.